addons/mass_mailing/wizard/mailing_contact_import.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, tools, _
+
+
+class MailingContactImport(models.TransientModel):
+    _name = 'mailing.contact.import'
+    _description = 'Mailing Contact Import'
+
+    @api.model
+    def default_get(self, fields_list):
```

Comment:
Would be better to directly give default_list_ids to the model. Don't like this kind of "active-based" hacks ... better use ORM standard if possible using default keys.